types/relay.go

```diff
@@ -18,7 +46,7 @@ type Relay struct {
 	PoktNodeAddress          string        `json:"poktNodeAddress"`
 	RelayStartDatetime       time.Time     `json:"relayStartDatetime"`
 	RelayReturnDatetime      time.Time     `json:"relayReturnDatetime"`
-	IsError                  bool          `json:"isError"`
+	IsError                  bool          `json:"isError"` // this field must be before the other error fields for validation to work
```

Comment:
I don't understand the comment, as this data structure is unordered, how it could be set before?

Reply:
The field are always reviewed in the order they are on the implementation